.github/workflows/appengine-deploy-it.yml

```diff
@@ -31,6 +31,18 @@ jobs:
         });
         server.listen(process.env.PORT || 8080);
         EOF
+    - id: build dependency
+      name: Build dependency
+      working-directory: setupGcloudSDK
```

Comment:
Should we add this to the run on push to path at the top of this file?

Reply:
Did you mean adding `'setupGcloudSDK**'` so that we test `appengine-deploy` for any change in`setupGcloudSDK`?

Reply:
done